Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -10,6 +10,12 @@ import WebKit
 import SwiftUI
 import Theme
 
+public let WebviewReloadNotification = "webviewReloadNotification"
```

Comment:
It can be moved to the Core/Extensions/Notification file to keep consistency.

<img width="927" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/127732735/a2a0960a-abc7-4b1e-a27d-fc6b6e44a0a0">

Reply:
Done